### ISSUE TITLE
Question or issue. Mutation happened, but not clear situation

### ISSUE THREAD

**Issue by garrotte13** [veden/Rampant#58]
Using the same old, manually "patched" gamesave with 79% evo.
Got a mutation happened (btw I'm unable to reproduce #57 anymore ! Is it good?), but I can't understand what particular happened and why. Here are screenshots:
1. Message is very strange. Did faction change occur (or alignment of faction?) ? What is "-672" ? And why it wrote about "2 of 6 mutations" ( I have default map setting value of 2 max mutations per faction)
![MessageToClarifyForDeathThresholdMutation](https://user-images.githubusercontent.com/95149904/144635017-74d9102f-a673-4fae-a692-80a14675adef.png)

2. Map position. South-West map corner beyond the sea. Very far away from my character position where a lot of biters were burnt by flame turrets - attacks were coming from North on my far away North wall.
![UnexpectedPlaceOfDeathThresholdMutation](https://user-images.githubusercontent.com/95149904/144635133-27637322-501f-4c3a-ad14-ed63a7f30a42.png)

**Comment by veden** [comment]
It has todo with the console message containing a nil somewhere it shouldn't.
Can you provide your save?

**Comment by garrotte13** [comment]
I played for about 20+ minutes to see such message. Not sure if it can be reproduced again. Here is the gamesave with provocation on North border:
[AtmoR4_test3.zip](https://github.com/veden/Rampant/files/7655039/AtmoR4_test3.zip)

I didn't make save after got that message above.

**Comment by garrotte13** [comment]
> Not sure if it can be reproduced again. 

Reproduced with this save. Just need to wait. Another message - this time thieves (probably because I used RA's personal cannon&shotgun - doing electro damage):
![image](https://user-images.githubusercontent.com/95149904/144833876-afb4c453-acc5-4802-a75c-bbf4d7887a30.png)

**Comment by veden** [comment]
This should be fixed in what is currently on the master branch

**Comment by garrotte13** [comment]
I was unable to get this message despite thousands of killed biters in over 30min playing with that particular gamesave. Perhaps need to kill even more and/or other restrictions apply. Maybe now it requires 11k killed biters generated by some fraction or something. I'll try to decrease coefficient in settings and see.

**Comment by garrotte13** [comment]
Rampant2.0.1
At 10k killled biters. 15% evo.
Became much easier to guess what happens
![image](https://user-images.githubusercontent.com/95149904/147396474-c6a80ced-4eb5-4222-a0b8-16423e331667.png)

Point on map is incorrect - there are no enemy structures or units in that point.
Mutation occured in many places.

**Comment by garrotte13** [comment]
Got this one on 25k biters killed. It's only 19% evolution, so I hope this mutation doesn't make real effect. I didn't find any nests of this faction
![image](https://user-images.githubusercontent.com/95149904/147509240-4bb48e4b-7fe6-48fb-be4a-bdced7d1f1c2.png)
As it is again about first mutation I assume there are many bases or "factions" or whatever objects mutation is applied to.

**Comment by veden** [comment]
Yes, there are many bases around the map. The location is the first place the base was created, so it will not always land perfectly on a group or area of spawners.
Around that point upto a certain distance will "belong" to that base.
The mutate from one faction to another is not instant and depending on the size of the base may never fully mutate to the new type.

**Comment by garrotte13** [comment]
>Around that location upto a certain distance will "belong" to that base.

First screenshot with mutation into Spawner Biter faction, it happened on two opposite ends of discovered map - far north and far south from map center. I do remember the message about that new faction mutation was shown only once. And a point on map was in the map center... what a magic coincidence. That's more of how homo sapience settled on planet Earth starting from Africa...

Anyway this new mechanics is really fun and challenging. Motivates to progress even when evolution is below my tech and industrial state.

**Comment by garrotte13** [comment]
And you were right about new faction to be found - I was wondering around the point listed in mutation message and found a new nest built with Regenerative biters faction. Hope it was the first and the last one :-) Meanwhile existing nests nearby didn't mutate like it happened with Spawner faction. Maybe it will occur later.

**Comment by garrotte13** [comment]
No, it was not the last. Massive mutation to Regenerative faction started slowly. Ok, with evo below 40% player still can survive with ease not forcing oil, gunturrets mk2, lasers and other.
I set mutations to happen at 1.5 coefficient of killed. For deathworld probably even 2.0 should be used. It's up to player and his gaming style. Thanks! Some action is always good.
I suppose in everything else it is Ok now. Will wait for 2nd mutation of the same "base" whatever it is, and bring up new issue if find anything.
